### PR TITLE
Refactor coinmarketcap crypto tests

### PR DESCRIPTION
**Pull request by cds95**

### CODE REVIEW

File: packages/sources/coinmarketcap/src/endpoint/globalMarketCap.ts [hunk 1]
```diff
@@ -27,6 +27,7 @@ export const execute: ExecuteWithConfig<Config> = async (request, _, config) =>
   response.data.result = Requester.validateResultNumber(response.data, [
     'data',
     'quote',
+    'quote',
```
Comment (cds95) [inline packages/sources/coinmarketcap/src/endpoint/globalMarketCap.ts:30]: Wanted to double check this.  I had to add this to get the adapter to work.  Is there an existing bug with this adapter?

File: packages/composites/token-allocation/src/config.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import Amberdata from '@chainlink/amberdata-adapter'
 import CoinApi from '@chainlink/coinapi-adapter'
 import CoinGecko from '@chainlink/coingecko-adapter'
-import CoinMarketCap from '@chainlink/coinmarketcap-adapter'
+import * as CoinMarketCap from '@chainlink/coinmarketcap-adapter'
```
Comment (cds95) [inline packages/composites/token-allocation/src/config.ts:4]: Had to update this as the variables are being exported differently from the coinmarketcap adapter.  Open to suggestions on a better way to do this.

File: packages/sources/coinmarketcap/src/endpoint/globalMarketCap.ts (hunk 1 above)
Comment (justinkaseman) [reply packages/sources/coinmarketcap/src/endpoint/globalMarketCap.ts:30]: It's working for me from the current `develop` branch :thinking:

File: packages/composites/token-allocation/src/config.ts (hunk 2 above)
Comment (justinkaseman) [reply packages/composites/token-allocation/src/config.ts:4]: Coinmarketcap looks the same as the other adapters.

What error was it hitting?

File: packages/sources/coinmarketcap/src/endpoint/globalMarketCap.ts (hunk 1 above)
Comment (cds95) [reply packages/sources/coinmarketcap/src/endpoint/globalMarketCap.ts:30]: I must have been looking at another endpoint's response.  Confirmed that this change is not needed.

File: packages/composites/token-allocation/src/config.ts (hunk 2 above)
Comment (cds95) [reply packages/composites/token-allocation/src/config.ts:4]: I had to change the way variables were exported to get the tests to work.

```
const server = expose(NAME, makeExecute(), undefined, endpointSelector).server

export { NAME, makeConfig, makeExecute, server }
```

**Review by justinkaseman — APPROVED** [review]

File: packages/composites/token-allocation/src/config.ts (hunk 2 above)
Comment (HenryNguyen5) [reply packages/composites/token-allocation/src/config.ts:4]: This looks like the correct way of importing to me, before with the `export =` syntax, TS was treating it as a "default export" which was one object that contained keys referencing the exports. Now with the `export {}` syntax, TS treats it as an ES module which has named exports like `NAME, makeConfig, makeExecute`. This is what results in the two different import styles we have now.

**Review by HenryNguyen5 — APPROVED** [review]
Looks great, nice job!

**Pull request merged**